### DISCUSSION TITLE
Pinned version `drf-spectacular==0.25.1` in `requirements/base.txt`

### DISCUSSION THREAD

**Discussion by blaketeres** [jokiefer/drf-spectecular-json-api#17] [General]
Nice package! It's a lifesaver. However, using this package is a bit tricky since `drf-spectacular` is pinned to an older version. Is this by design? I hack-installed this with the latest version of `drf-spectacular==1.26.5`, and it still worked. Is it possible to open up the range of allowed versions here? Thanks

**Comment by jokiefer** [comment]
thx for your report!

Sure, it is possible, but keep in mind that the sem version of the parent project is still under <1. So there can anything happening from version to version.

I will open it and add matrix test cases.

**Comment by jokiefer** [comment]
version [v0.4.0](https://pypi.org/project/drf-spectacular-jsonapi/) is now available which supports drf-spectacular>=0.25.x.

Have fun with it ;)

**Reply by blaketeres** [reply]
Thank you much!
